autoload/neomake/utils.vim

```diff
@@ -68,7 +68,9 @@ function! neomake#utils#WideMessage(msg) abort " {{{2
     "convert tabs to spaces so that the tabs count towards the window
     "width as the proper amount of characters
     let chunks = split(msg, "\t", 1)
+    " vint: -ProhibitUnnecessaryDoubleQuote
     let msg = join(map(chunks[:-2], 'v:val . repeat(" ", &tabstop - strwidth(v:val) % &tabstop)'), '') . chunks[-1]
+    " vint: +ProhibitUnnecessaryDoubleQuote
```

Comment:
That was my problem, disabling/enabling vint checks through comments, then leaves something wrong when running vint --error. I thought that ' ' were necessary everywhere, but your suggestion seems to be working fine, thanks!